src/StreamFactoryInterface.php

```diff
@@ -48,5 +16,5 @@ public function createStreamFromResource($body);
      *
      * @return StreamInterface
      */
-    public function createStreamFromString($body);
+    public function createStream($body = '');
```

Comment:
Hey @shadowhand ,

Slim can of-course add the feature if the PSR is published, so I hope that will not make a BC break. Aren't we looking forward or are we trying to solve the problems with only two for the current time ?